packages/angular/cli/src/analytics/analytics.ts

```diff
@@ -12,10 +12,10 @@ import * as inquirer from 'inquirer';
 import { v4 as uuidV4 } from 'uuid';
 import { colors } from '../utilities/color';
 import { getWorkspace, getWorkspaceRaw } from '../utilities/config';
+import { analyticsDisabled, analyticsShareDisabled } from '../utilities/environment-options';
```

Comment:
Nit: Consider moving the environment options changes into a separate refactor commit.